src/MongoDB.Driver.Core/MongoIncompatibleDriverException.cs

```diff
@@ -63,6 +62,13 @@ private static string FormatMessage(ClusterDescription description)
             return $"Server at {EndPointHelper.ToString(incompatibleServer.EndPoint)} requires wire version {incompatibleServer.WireVersionRange.Min},"
                 + $" but this version of the driver only supports up to {Cluster.SupportedWireVersionRange.Max}.";
         }
+
+        private static bool IsServerIncompatible(ServerDescription description)
+        {
+            return description.Type != ServerType.Unknown &&
+                   description.WireVersionRange != null &&
+                   !description.WireVersionRange.Overlaps(Cluster.SupportedWireVersionRange);
```

Comment:
We don't do vertical alignment. Using consistent indentation this could be formatted as:

    return 
        description.Type != ServerType.Unknown &&
        description.WireVersionRange != null &&
        !description.WireVersionRange.Overlaps(Cluster.SupportedWireVersionRange);

Reply:
Done